### PR TITLE
Config file for core db finalisation

### PR DESCRIPTION
**Pull request by leannehaggerty**
# Requirements
When creating your Pull request, please fill out the template below:

# PR details
New feature
Core db finalisation sub-pipeline
https://www.ebi.ac.uk/panda/jira/browse/ENSGENEBUI-480

# Testing
perl -c LastZ.pm - no issues
init_pipeline - creates pipeline that looks correct

# Assign to the weekly GitHub reviewer
_If you are a member of Ensembl, please check the Genebuild weekly Rotas and assign this week's GitHub reviewer to the PR_

### CODE REVIEW

File: modules/Bio/EnsEMBL/Analysis/Hive/Config/FinaliseCoreDB.pm [hunk 1]
```diff
@@ -0,0 +1,570 @@
...
+    'protein_entry_loc'  => catfile( $self->o('base_blast_db_path'), 'uniprot', $self->o('uniprot_version'), 'entry_loc' ),    # Used by genscan blasts and optimise daf/paf. Don't change unless you know what you're doing
+
+########################
+# Pipe and ref db info
+########################
+
+    'projection_source_db_name'         => '',                                                                                 # This is generally a pre-existing db, like the current human/mouse core for example
+    'projection_source_db_server'       => 'mysql-ens-mirror-1',
+    'projection_source_db_port'         => '4240',
+    'projection_source_production_name' => '',
```
Comment (thibauthourlier) [inline modules/Bio/EnsEMBL/Analysis/Hive/Config/FinaliseCoreDB.pm:81]: This is not used in the pipeline and can be removed

File: modules/Bio/EnsEMBL/Analysis/Hive/Config/FinaliseCoreDB.pm [hunk 2]
```diff
@@ -0,0 +1,570 @@
...
+      -module     => 'Bio::EnsEMBL::Analysis::Hive::RunnableDB::HiveAnalysisSanityCheck',
+      -parameters => {
+        skip_rnaseq                => $self->o('skip_rnaseq'),
+        skip_projection            => $self->o('skip_projection'),
+        target_db                  => $self->o('reference_db'),
+        sanity_check_type          => 'gene_db_checks',
+        min_allowed_feature_counts => get_analysis_settings( 'Bio::EnsEMBL::Analysis::Hive::Config::SanityChecksStatic',
+          'gene_db_checks' )->{ $self->o('uniprot_set') }->{'core'},
+      },
+
```
Comment (thibauthourlier) [inline modules/Bio/EnsEMBL/Analysis/Hive/Config/FinaliseCoreDB.pm:512]: Coud this blank line please be removed?

**Review by thibauthourlier — CHANGES_REQUESTED** [review]

File: modules/Bio/EnsEMBL/Analysis/Hive/Config/FinaliseCoreDB.pm (hunk 1 above)
Comment (leannehaggerty) [reply modules/Bio/EnsEMBL/Analysis/Hive/Config/FinaliseCoreDB.pm]: Ah yes, only needed the db name.
Done

File: modules/Bio/EnsEMBL/Analysis/Hive/Config/FinaliseCoreDB.pm (hunk 2 above)
Comment (leannehaggerty) [reply modules/Bio/EnsEMBL/Analysis/Hive/Config/FinaliseCoreDB.pm]: Yep

**Review by thibauthourlier — APPROVED** [review]

**Pull request merged**